R/R-igraph/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           R 1.0
+
+R.setup             cran igraph igraph 1.3.5
+revision            0
+maintainers         {gmail.com:szhorvat @szhorvat} {@barracuda156 gmail.com:vital.had} openmaintainer
+license             {GPL-2 GPL-3}
+description         Network Analysis and Visualization
+long_description    Routines for simple graphs and network analysis. \
+                    It can handle large graphs very well and provides functions \
+                    for generating random and regular graphs, graph visualization, \
+                    centrality methods and much more.
+homepage            https://igraph.org
+checksums           rmd160  dc3bfb06c279d7b4f7735b715c19e324aafc8223 \
+                    sha256  9e615d67b6b5b57dfa54ec2bbc8c29da8f7c3fe82af1e35ab27273b1035b9bd4 \
+                    size    2499723
+
+depends_build-append \
+                    port:pkgconfig
```

Comment:
I _think_ `pkgconfig` is not required, but let's leave it in for now, just in case. I'll remove it at the next update if I have the time to verify that it's truly not needed. Version 1.4.0 is coming soon.

Reply:
Ok, cool. Other three suggestions I have implemented now and rebased the branch.